index.html

```diff
@@ -10,5 +10,35 @@
     <header class="header">
         <h1>Hello world</h1>
     </header>
+    <form action="" method="post">
+        <ul>
+            <li>
+                <label for="name">Name:</label>
+                <input type="text" name="user_name" id="name" required>
+            </li><br>
+            <li>
+                <label for="email">E-mail:</label>
+                <input type="email" name="user_email" id="email" required>
+            </li><br>
+            <li>
+                <label for="message">Message:</label>
+                <textarea name="user_message" id="message" cols="30" rows="10" placeholder="Type your comment here..." required></textarea>
```

Comment:
You should add the attribute maxlength to the inputs to choose the maximum characters allowed ^^